src/main/java/gov/usgs/wma/waterdata/collections/CollectionsController.java

```diff
@@ -135,58 +149,61 @@ public String getOgcCollection(@RequestParam(value = "f", required = false, defa
 	@GetMapping(value = "collections/{collectionId}/items", produces = MediaType.APPLICATION_JSON_VALUE)
 	public String getOgcCollectionFeatures(
 		HttpServletRequest request,
-		@Size(min=0, max=250, message="The number of countries is not between {min} and {max} occurrences")
-		@Parameter(description="Countries in 2 digit FIPS format (for example, US or MX)")
-		@RequestParam(value="country", required = false) List<@Pattern(regexp=REGEX_FIPS_COUNTRY) String> countries,
+		@Size(min=1, max=250, message="The number of countries queried on must be between {min} and {max}")
+		@Parameter(description=FIPS_COUNTRY_DESC)
+		@RequestParam(value="countryFIPS", required = false) List<@Pattern(regexp=REGEX_FIPS_COUNTRY, message=REGEX_FIPS_COUNTRY_MESS) String> countries,
 
-		@Size(min=0, max=1000, message="The number of counties is not between {min} and {max} occurrences")
-		@Parameter(description="Counties in FIPS format.  For example, Dane County, Wisconsin would be 55025")
-		@RequestParam(value="county", required = false) List<@Pattern(regexp=REGEX_FIPS_COUNTY) String> counties,
+		@Size(min=1, max=1000, message="The number of counties queried on must be between {min} and {max}")
+		@Parameter(description=FIPS_COUNTY_DESC)
+		@RequestParam(value="countyFIPS", required = false) List<@Pattern(regexp=REGEX_FIPS_COUNTY, message=REGEX_FIPS_COUNTY_MESS) String> counties,
 
-		@Size(min=0, max=100, message="The number of states is not between {min} and {max} occurrences")
-		@Parameter(description="States in FIPS format.  For example, Wisconsin would be 55")
-		@RequestParam(value="state", required = false) List<@Pattern(regexp=REGEX_FIPS_STATE) String> states,
+		@Size(min=1, max=55, message="The number of states queried on must be between {min} and {max}")
+		@Parameter(description=FIPS_STATE_DESC)
+		@RequestParam(value="stateFIPS", required = false) List<@Pattern(regexp=REGEX_FIPS_STATE, message=REGEX_FIPS_STATE_MESS) String> states,
 
-		@Size(min=0, max=1000, message="The number of hydrological units is not between {min} and {max} occurrences")
+		@Size(min=1, max=1000, message="The number of hydrologic units queried on must be between {min} and {max}")
 		@Parameter(description="For example, 040103020107")
-		@RequestParam(value="hydrologicalUnit", required = false)
-			List<@Pattern(regexp=REGEX_FIPS_HYDRO) String> hydrologicalUnits,
+		@RequestParam(value="hydrologicUnit", required = false)
+			List<@Pattern(regexp=REGEX_FIPS_HYDRO, message=REGEX_FIPS_HYDRO_MESS) String> hydrologicUnits,
 		@Parameter(description="For example, N9999OTHER")
+
 		@RequestParam(value="nationalAquiferCode", required = false) String nationalAquiferCode,
 		@Parameter(description="For example, USGS-343204093005501")
```

Comment:
Do you need to remove USGS here for the description of monitoringLocationNumber?  You removed USGS in the unit test.

Reply:
you do want to drop the `USGS-` which was from when we were querying on identifier instead of number